### PR TITLE
Better display

### PR DESCRIPTION
**Pull request by EirinSognnes**

### CODE REVIEW

File: app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import android.view.LayoutInflater;
 import com.BetterTogether.app.Fragments.UserListFragment;
 
+import com.BetterTogether.app.R;
```
Comment (JensGM) [inline app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java]: R?

Comment (EirinSognnes) [reply app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java]: Android things, it's for "Resources". Need it for adding a colour theme to popupView  (R.style.MyTheme).

Comment (JensGM) [reply app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java]: Is this an android naming conversion thing?

Comment (EirinSognnes) [reply app/src/main/java/com/BetterTogether/app/Dialogs/PopupView.java]: It's not mine. It comes from Android, so I can't do anything about the name.

**Pull request merged**